test/cpp/util/test_credentials_provider.h

```diff
@@ -44,6 +44,22 @@ namespace grpc {
 namespace testing {
 
 const char kInsecureCredentialsType[] = "INSECURE_CREDENTIALS";
+const char kTlsCredentialsType[] = "TLS_CREDENTIALS";
```

Comment:
I put it in the header file because all of the types have to be unique, so they are part of the interface.